appveyor.yml

```diff
@@ -1,20 +1,33 @@
 environment:
   SAXON_CP: '%TEMP%\xspec\saxon\saxon9he.jar'
   matrix:
-    # latest Saxon 9.7 version
+    # latest Saxon 9.7 version and XML Calabash
     - SAXON_VERSION: 9.7.0-15
+      XMLCALABASH_VERSION: 1.1.15-97

```

Comment:
Sorry for the confusion!

Yes, to save time, the AppVeyor configuration is slightly different from Travis. If `XMLCALABASH_VERSION` is not defined, two things happen:

* `appveyor.yml` skips downloading XML Calabash.
* The test script `xspec-bat.cmd` skips the XProc test and **treats the test as Success**.

I updated the test script just now. The test script no longer treats the skipped test as Success. See [the updated build logs](https://ci.appveyor.com/project/xspec/xspec/build/1.0.115).
The skipped test has a gray (not green) sidebar on [AppVeyor control panel](https://ci.appveyor.com/project/xspec/xspec/build/1.0.115/job/yq23ul6gngdxgdok/tests). Also [the log](https://ci.appveyor.com/project/xspec/xspec/build/1.0.115/job/yq23ul6gngdxgdok) prints 'SKIP'.